pkg/kubelet/kubelet.go

```diff
@@ -2604,6 +2604,7 @@ func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 				// Check if we can admit the pod; if not, reject it.
 				if ok, reason, message := kl.canAdmitPod(activePods, pod); !ok {
 					kl.rejectPod(pod, reason, message)
+					metrics.AdmissionRejectionsTotal.WithLabelValues(reason).Inc()
```

Comment:
This should also be logged in the previous case (line 2595). It might be better to just move the recording to the `rejectPod` method.